task/07-yield-tasks.js

```diff
@@ -145,7 +205,8 @@ function* mergeSortedSequences(source1, source2) {
  *   Most popular implementation of the logic in npm https://www.npmjs.com/package/co
  */
 function async(generator) {
-    throw new Error('Not implemented');
+  let co = require('co');
+  return co(generator);
```

Comment:
The idea is to implement alternative solution based on generators and Promises, I'll reopen th PR

Reply:
ok, done.